### PR TITLE
BE: Tags Cannot Be Blank

### PR DESCRIPTION
**Pull request by jared23m**
added a check on create and edit recipe that prevents blank tags from being added

### CODE REVIEW

File: BE/api/recipes.js [hunk 1]
```diff
@@ -24,7 +24,20 @@ recipesRouter.get("/", async (req, res, next) => {
 recipesRouter.post("/", requireAdmin, async (req, res, next) => {
   const { id: userId } = req.user;
   const { body } = req;
-
+  
+  if(body.tags && body.tags != []){
```
Comment (ljreaux) [inline BE/api/recipes.js:28]: I'm not sure this works.

Comment (jared23m) [reply BE/api/recipes.js:28]: why

Comment (jared23m) [reply BE/api/recipes.js:28]: which part? i tested

Comment (ljreaux) [reply BE/api/recipes.js:28]: I thought there were issues testing something against a blank array.

Comment (jared23m) [reply BE/api/recipes.js:28]: right now if the recipe has tags, and you edit tags and put in an empty array field, the tags do not update. But that is an extreme edge case and I'm not willing to work on that. This condition to check for blank tags works though.

Comment (jared23m) [reply BE/api/recipes.js:28]: im dumb im editing it

**Review by amead99 — APPROVED** [review]

**Review by BarryFullStack — APPROVED** [review]

**Pull request merged**